ui/packages/catalog-ui-search/src/main/webapp/component/search-form/forms-sharing/search-form-sharing.collection.js

```diff
@@ -84,11 +84,19 @@ let bootstrapPromise = sharedSearchFormPromise();
     };
    },
    checkIfShareable: function(template) {
+       if(this.checkIfOwner(template)){
```

Comment:
I think your spacing is inconsistent. Might want to pass this through Standard.js, I think that is the norm now.